iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/TopologyModificationUtils.java

```diff
@@ -439,12 +439,14 @@ public static Set<Integer> getFeederPositions(VoltageLevel voltageLevel) {
      */
     public static Map<String, List<Integer>> getFeederPositionsByConnectable(VoltageLevel voltageLevel) {
         Map<String, List<Integer>> feederPositionsOrders = new HashMap<>();
-        voltageLevel.getConnectables().forEach(connectable -> {
-            ConnectablePosition<?> position = (ConnectablePosition<?>) connectable.getExtension(ConnectablePosition.class);
-            if (position != null) {
-                List<Integer> orders = getOrderPositions(position, voltageLevel, connectable, false, Reporter.NO_OP);
-                feederPositionsOrders.put(connectable.getId(), orders);
+        Map<String, List<ConnectablePosition.Feeder>> feedersByConnectable = getFeedersByConnectable(voltageLevel);
+        feedersByConnectable.forEach((k, v) -> {
```

Comment:
You can directly do `getFeedersByConnectable(voltageLevel).forEach(...` without using an intermediary variable.